cruise.umple/test/cruise/umple/compiler/UmpleParserTest.java

```diff
@@ -1428,6 +1428,16 @@ public void ReqSameNameCombine()
           assertNoWarningsParse("450_ReqSameOneBlank.ump");
           assertNoWarningsParse("450_ReqSameLang.ump");
   }
+  
+   @Test
+  public void ReqSort()
```

Comment:
This testset just specfies that the files parse. There is no testing of the output to verify that the sort order is correct